reana_client/cli/workflow.py

```diff
@@ -81,7 +81,11 @@ def workflow_execution_group(ctx):
     help="Get output in JSON format.",
 )
 @click.option(
-    "--all", "show_all", count=True, help="Show all workflows including deleted ones."
```

Comment:
Can you please add a line to the `CHANGES.rst` about this functionality change, for example:

- Changes ``list`` command to include deleted workflows by default.